src/Compilers/CSharp/Test/Syntax/Parsing/LambdaAttributeParsingTests.cs

```diff
@@ -2206,41 +2206,42 @@ public void NullableType_Is_01()
         public void NullableType_Is_02()
         {
             string source = "x is string ? [A] y";
-            UsingExpression(source);
+            UsingExpression(source,
+                // (1,19): error CS1003: Syntax error, ':' expected
+                // x is string ? [A] y
+                Diagnostic(ErrorCode.ERR_SyntaxError, "y").WithArguments(":").WithLocation(1, 19));
```

Comment:
note: this change is appropriate.  it's not legal to say `x is string?[A]`.  `string?[A]` is *not* a type.  While you can instantiate a `new string?[A]`, it's type is now `string?[]`.  So this now properly becomes a case where this is a conditional that creates a collection, but is missing a colon.